pkgs/applications/networking/mailreaders/evolution/evolution/default.nix

```diff
@@ -44,11 +44,11 @@
 
 stdenv.mkDerivation rec {
```

Comment:
How about taking the opportunity to edit this file to add a commit and switch to the `finalAttrs` pattern ?

Let me share a couple of useful links regarding this:

1. History: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293
2. Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes
3. Recent example of implementation: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe

Please make this change in a new commit.

Cheers.

Reply:
Hi, I am not against this but I would likely not bump anything if I am requested to do
 this on every bumps :-( Do you mind open your PR to address the changes?

Reply:
I tend to do this when bumping major versions, probably will do the change for all gnome packages in GNOME 45 PR. Doing it for minor ones will make backporting annoying.

Reply:
Ok ! Message understood. I will adapt my comments for the next time.

Reply:
> I doubt I will do it myself at some point

Hmmm okay I will reopen this then since this sounds like some emergency bugfix (one week since last upd I think), sorry I am actually not that interested (i.e. quite lazy) in specific things

Reply:
It's perfectly fine, I will also pay attention next time :)

Thanks for reopening it ! Much appreciated.